### ISSUE TITLE
[COGNITION]: Can we review the language for Community Care appointment review times?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6279]
## Description

I was noticing while auditing today that our review screen for Community Care appointments have a text snippet "...in the evening" for preferred appointment time. I am attaching a screen shot and wondering if that should be "afternoon" or some other language.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* [ ] As a screen reader user, I want to understand clearly if my preferred appointment time is in the afternoon, or the evening. I understand evening to be dinner time (approx. 6pm local time) and after.

## WCAG or Vendor Guidance (optional)

* [Unusual Words: Understanding SC 3.1.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-idioms.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-27 at 11.57.22 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/3fd09c9c-757a-48b0-bc86-0d38bacac6aa)

**Comment by jenniemc** [comment]
Hi @lenaecb, I can pick this ticket up but want to make sure it doesn't need to go through Peggy for word-smithing.

**Comment by lenaecb** [comment]
Thanks @jenniemc - let me check

@peggygannon - what are your thoughts on evening vs. afternoon? After going through the flow just now, I would also assume that we would want this to say 'afternoon' as opposed to evening like it does now.

**Comment by jenniemc** [comment]
Hi @peggygannon, can we get your eyes on this ticket concerning use of "evening" vs "afternoon"? We're currently using "evening" which may be misleading for users.

cc @lenaecb

**Comment by peggygannon** [comment]
Hello, @lenaecb @jenniemc Hello, sorry I missed this one. 

Yes, we should say Afternoon rather than evening.
